tests/test_yaml2sbml/ode_input_empty_section.yaml

```diff
@@ -0,0 +1,106 @@
+time:
+    variable: t
+
+parameters:
+
+    - parameterId: c1
+      nominalValue: 1
+
+      parameterScale: lin
+      lowerBound: 0
+      upperBound: 10
+      estimate: 0
+
+    - parameterId: Shalve
+      nominalValue: 0.1
+
+      parameterScale: log10
+      lowerBound: 0.1
+      upperBound: 10
+      estimate: 1
+
+    - parameterId: Vh
+      nominalValue: 0.1
+
+      parameterScale: lin
+      lowerBound: 0
+      upperBound: 50
+      estimate: 0
+
+    - parameterId: h
+      nominalValue: 0.1
+
+      parameterScale: lin
+      lowerBound: 0
+      upperBound: 10
+      estimate: 0
+
+    - parameterId: Vmm
+      nominalValue: 0.1
+
+      parameterScale: lin
+      lowerBound: 0
+      upperBound: 10
+      estimate: 0
+
+    - parameterId: Km
+      nominalValue: 0.1
+
+      parameterScale: lin
+      lowerBound: 0
+      upperBound: 10
+      estimate: 0
+
+    - parameterId: v1
+      nominalValue: 0.1
+
+      parameterScale: lin
+      lowerBound: 0
+      upperBound: 10
+      estimate: 0
+
+    - parameterId: k4
+      nominalValue: 0.1
+
+      parameterScale: lin
+      lowerBound: 0
+      upperBound: 10
+      estimate: 0
+
+
+
+functions:
+    - functionId: hill
+      arguments: V, s, Shalve, h
+      formula: V * (s/Shalve)^h / (1 + (s/Shalve)^h )
+
+    - functionId: MM
+      arguments: V, s, Km
+      formula: V*s / (Km + s)
+
+odes:
+    - stateId: S1
+      rightHandSide: v1 - MM(S1, Vmm, Km)*c1^2
+      initialValue: 0.1
+
+    - stateId: S2
+      rightHandSide: MM(S1, Vmm, Km) - hill(S2, Shalve, Vh, h)
+      initialValue: 1
+
+    - stateId: S3
+      rightHandSide: hill(S2, Shalve, Vh, h) - k4*S3 + log10(c1)
+      initialValue: 1
+
+
+observables:
```

Comment:
Why does it have an empty observable section?

Reply:
Ah, got it :D

Reply:
Failing test xD